### ISSUE TITLE
Office: Remove DDE relations

### ISSUE THREAD

**Issue by PunKeel** [docbleach/DocBleach#30]
https://pwndizzle.blogspot.fr/2017/03/office-document-macros-ole-actions-dde.html

Samples (they both open calc.exe)

[DDE.xlsx](https://github.com/docbleach/DocBleach/files/1380113/DDE.xlsx)
[DDE.xls.zip](https://github.com/docbleach/DocBleach/files/1380115/DDE.xls.zip)

```
[main] TRACE xyz.docbleach.module.ooxml.OOXMLBleach - Part name: /xl/externalLinks/externalLink1.xml
[main] DEBUG xyz.docbleach.module.ooxml.OOXMLBleach - Content type: application/vnd.openxmlformats-officedocument.spreadsheetml.externalLink+xml for part /xl/externalLinks/externalLink1.xml


[main] DEBUG xyz.docbleach.module.ooxml.OOXMLBleach - Relation type 'http://schemas.openxmlformats.org/officeDocument/2006/relationships/externalLink' found from 'Name: /xl/workbook.xml - Content Type: application/vnd.openxmlformats-officedocument.spreadsheetml.sheet.main+xml' to '/xl/externalLinks/externalLink1.xml'
```

Content of `externalLink1.xml`: 
```<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
<externalLink xmlns="http://schemas.openxmlformats.org/spreadsheetml/2006/main" xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006" mc:Ignorable="x14" xmlns:x14="http://schemas.microsoft.com/office/spreadsheetml/2009/9/main"><ddeLink xmlns:r="http://schemas.openxmlformats.org/officeDocument/2006/relationships" ddeService="cmd" ddeTopic="/c calc"><ddeItems><ddeItem name="A0" advise="1"/><ddeItem name="StdDocumentName" ole="1" advise="1"/></ddeItems></ddeLink></externalLink>```

I haven't found a clean way to remove it for the moment. Fix might look like what's discussed in #28

**Comment by PunKeel** [comment]
https://twitter.com/ParvezGHH/status/918423761086492672

**Comment by PunKeel** [comment]
Fix'd by #31
